R/zzz.R

```diff
@@ -1,3 +1,5 @@
+#' @importFrom usethis edit_file
```

Comment:
Interesting. I've not seen folks moving dependencies away from the functions they are called in. To me that has the potential of accidentally removing a dependency (import) when you think you no longer need it and cluttering up another location 🤔

Reply:
Here it is in [waldo ](https://github.com/r-lib/waldo/blob/master/R/waldo-package.R)and [dplyr](https://github.com/tidyverse/dplyr/blob/master/R/dplyr.r).  I think it mainly helps with redundancy and reducing namespace conflicts within the package itself. 

I agree there are pros and cons and I can switch it back if people disagree with this approach.

Reply:
in dplyr `@importFrom` is littered across the R code. I can see value in cases that there have been namespace conflicts identified calling the particulars (a la dplyrs issue with `vctr::data_frame()` vs `tibble::data_frame()`), and we resolve as such. but personally I think it is much clearer to list the imports for your function where they are needed.

definitely interested to hear @mariev 's interpretation

Reply:
I think both paradigms are technically legitimate, but keeping `@importFrom ` with function level documentation is a better fit for review and long-term maintenance of a package, for the reasons that @thebioengineer stated initially. 

Definitely don't use a mix of the two in a single package!

Reply:
Works for me. importFrom has been moved back to the function documentation